pkg/pillar/cmd/upgradeconverter/upgradeconvertor_test.go

```diff
@@ -0,0 +1,37 @@
+package upgradeconverter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestHandleUpgradeGlobalConfig(t *testing.T) {
+	runPath, _ := filepath.Abs(filepath.Dir(os.Args[0]))
+	OldVersion = runPath + "/oldconfig/global.json"
+	NewVersion = runPath + "/oldconfigV2/global.json"
+	oldConfig, _ := json.Marshal(OldGlobalConfig{})
```

Comment:
Hmm.. I think I did not understand how u r unit-testing this when the file paths are hard coded as constants in the code above??